styles/styles.css

```diff
@@ -223,26 +160,28 @@ main img {
   width: 100%;
 }
 
-/* sections */
+/* sections: 56px spacing at the bottom */
 main .section {
-  padding: 64px 16px;
+  padding-bottom: 56px;
 }
 
-@media (width >= 600px) {
-  main .section {
-    padding: 64px 32px;
-  }
+main .section > div:not(.hero-wrapper) {
```

Comment:
Probably better to have `hero` related styles/overrides in hero block's css itself. One option could be to have the hero.css override this instead of the `not` selector here.

Reply:
+1